seam/seam_multi_workspace.py

```diff
@@ -36,6 +36,8 @@ def __init__(
         *,
         endpoint: Optional[str] = None,
         wait_for_action_attempt: Optional[Union[bool, Dict[str, float]]] = False,
+        client: Optional[requests.Session] = None,
+        client_options: Optional[Dict[str, Any]] = None,
```

Comment:
Let's remove these for now like in the other PR.

Reply:
https://github.com/seamapi/python-next/pull/63/commits/8ba79cfccfbb02571f183e01ba00d45183984d2c